core/src/main/kotlin/com/google/container/tools/core/settings/UsageTrackerConfigurableProvider.kt

```diff
@@ -32,6 +34,21 @@ class UsageTrackerConfigurableProvider : ConfigurableProvider() {
      * Only create the menu item if usage tracking is available. For example, if running in dev
      * mode with no analytics ID environment variable configured, hide the usage track menu item.
      */
-    override fun canCreateConfigurable(): Boolean =
-        UsageTrackerManagerService.instance.isUsageTrackingAvailable()
+    override fun canCreateConfigurable(): Boolean {
+        /**
+         * This check is a workaround for the fact that the older versions of the GCT plugin do not
+         * have an ID configured for this panel. Since we only want a single instance of this panel
+         * to appear for all installed Google plugins, this checks to ensure that it was not already
+         * registered by the GCT plugin. Once all users migrate to newer versions of the GCT plugin,
+         * we can remove this. For this to work, both provider classes have to have the same name.
+         */
+        val canCreateConfigurable: Boolean =
+            Configurable.APPLICATION_CONFIGURABLE.extensionList.filter {
```

Comment:
To be clear, this will get a complete list of all configurables for the whole IDE and then you check the plugin is alone (meaning it's the only one that registered the provider). With Cloud Tools present, the count is 2, correct?

Reply:
exactly

Reply:
if you feel the javadoc isn't clear enough I can update?

Reply:
No I think the javadoc is great, I was just making sure the code reads correct (I looked up `APPLICATION_CONFIGURABLE` quickly), I think this is perfectly clear.